packages/frontend/src/components/RichTextEditor/MenuBar.scss

```diff
@@ -0,0 +1,30 @@
+.divider {
+  background-color: rgba(#fff, 0.25);
+  height: 1.25rem;
+  margin-left: 0.5rem;
+  margin-right: 0.75rem;
+  width: 1px;
+}
+
+.menu-item {
+  background-color: transparent;
+  border: none;
+  border-radius: 0.4rem;
+  color: #fff;
+  cursor: pointer;
+  height: 1.75rem;
+  margin-right: 0.25rem;
+  padding: 0.25rem;
+  width: 1.75rem;
+
+  svg {
+    fill: currentColor;
+    height: 100%;
+    width: 100%;
+  }
+
+  &:hover,
+  &.is-active {
+    background-color: #303030;
+  }
+}
```

Comment:
Nit

```suggestion
    &.is-disabled {
      color: #aa004b;
      cursor: default;
    }
}
```

Small feature request to add `isDisabled`, ignore if you think too heavy

Reply:
Actually nvm, I looked at this for a bit. A bit tricky, not worth.

e.g. `deleteColumn` is a no-op if there is only 1 column, so we want to disable that. But there doesnt' seem to be an easy way to get current column count.